projects/rocprofiler-systems/source/lib/rocprof-sys/library/components/category_region.hpp

```diff
@@ -28,17 +28,89 @@
 #include "core/timemory.hpp"
 #include "library/causal/data.hpp"
 #include "library/runtime.hpp"
+#include "library/thread_info.hpp"
 #include "library/tracing.hpp"
 #include "library/tracing/annotation.hpp"
 
+#include <map>
+#include <thread>
 #include <timemory/components/gotcha/backends.hpp>
 #include <timemory/hash/types.hpp>
 #include <timemory/mpl/concepts.hpp>
 #include <timemory/mpl/types.hpp>
 #include <timemory/utility/types.hpp>
 
+#include "core/trace_cache/cache_manager.hpp"
+
 #include <string_view>
 
+#include <utility>
+
+namespace
+{
+
+void
+cache_region(uint64_t thread_id, const std::string& name, uint64_t start_ts,
+             uint64_t end_ts, const std::string& category)
+{
+    size_t zero = 0;
```

Comment:
```suggestion
   constexpr size_t NO_CORRELATION_ID = 0;
```
Maybe this would be more descriptive and easier to understand why it is hard-coded to 0.